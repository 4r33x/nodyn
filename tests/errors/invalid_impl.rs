use nodyn::nodyn;

nodyn! {
    pub enum Value {
        i32,
    }
    impl InvalidFeature; // Unknown feature
}

fn main() {}
