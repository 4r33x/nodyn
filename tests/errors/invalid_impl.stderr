error: expected curly braces
 --> tests/errors/invalid_impl.rs:7:24
  |
7 |     impl InvalidFeature; // Unknown feature
  |                        ^
